test/arbitrum/JPEGzMainnetOracle.t.sol

```diff
@@ -0,0 +1,505 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.5;
+
+import "forge-std/Test.sol";
+import "../../contracts/ETHVaultHandler.sol";
+import "../../contracts/Orchestrator.sol";
+import "../../contracts/TCAP.sol";
+import "../../contracts/oracles/ChainlinkOracle.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+
+contract JPEGzMainnetTest is Test {
+  // events
+  event NewMinimumTCAP(address indexed _owner, uint256 _minimumTCAP);
+  event NewMintFee(address indexed _owner, uint256 _mintFee);
+  event NewBurnFee(address indexed _owner, uint256 _burnFee);
+
+  // Setup
+  ETHVaultHandler ethVault;
+  Orchestrator orchestrator = new Orchestrator(address(this));
+  TCAP jpegz = new TCAP("JPEGz Token", "JPEGz", 0, (orchestrator));
+  AggregatorV3Interface jpegzAggregator =
+    AggregatorV3Interface(0x8D0e319eBAA8DF32e088e469062F85abF2eBe599);
+  AggregatorV3Interface ethAggregator =
+    AggregatorV3Interface(0x639Fe6ab55C921f74e7fac1ee960C0B6293ba612);
+  ChainlinkOracle jpegzOracle =
+    new ChainlinkOracle(address(jpegzAggregator), address(this));
+  ChainlinkOracle ethOracle =
+    new ChainlinkOracle(address(ethAggregator), address(this));
+  address weth = 0x82aF49447D8a07e3bd95BD0d56f35241523fBab1;
+  address user = address(0x1);
+  address user2 = address(0x2);
+
+  //// Params
+  uint256 divisor = 1000000000;
+  uint256 ratio = 150;
+  uint256 burnFee = 150;
+  uint256 mintFee = 150;
+  uint256 liquidationPenalty = 20;
+  address treasury = address(0x3);
+
+  function setUp() public {
+    ethVault = new ETHVaultHandler(
+      orchestrator,
+      divisor,
+      ratio,
+      burnFee,
+      mintFee,
+      liquidationPenalty,
+      address(jpegzOracle),
+      jpegz,
+      address(weth),
+      address(ethOracle),
+      address(ethOracle),
+      treasury,
+      0 ether
+    );
+
+    orchestrator.addTCAPVault(jpegz, ethVault);
+  }
+
+  function testConstructor_ShouldSetParams_WhenInitialized() public {
+    assertEq(address(orchestrator), ethVault.owner());
+    assertEq(divisor, ethVault.divisor());
+    assertEq(ratio, ethVault.ratio());
+    assertEq(burnFee, ethVault.burnFee());
+    assertEq(burnFee, ethVault.mintFee());
+    assertEq(liquidationPenalty, ethVault.liquidationPenalty());
+    assertEq(address(jpegzOracle), address(ethVault.tcapOracle()));
+    assertEq(address(jpegz), address(ethVault.TCAPToken()));
+    assertEq(address(weth), address(ethVault.collateralContract()));
+    assertEq(address(ethOracle), address(ethVault.collateralPriceOracle()));
+    assertEq(address(ethOracle), address(ethVault.ETHPriceOracle()));
+    assertEq(treasury, ethVault.treasury());
+    assertEq(0 ether, ethVault.minimumTCAP());
+  }
+
+  function testConstructor_ShouldRevert_WhenBurnFeeIsHigh(uint256 _burnFee)
+    public
+  {
+    if (_burnFee > 1000) {
```

Comment:
Forge has a trick for fuzzing: `vm.assume(_burnFee > 1000)`.
With this trick, you should be able to get rid of the if block.